ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py

```diff
@@ -0,0 +1,61 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy
+import ivy_tests.test_ivy.helpers as helpers
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# prod
+@handle_cmd_line_args
+@given(
+    dtype_and_x=np_frontend_helpers.dtype_x_bounded_axis(available_dtypes=ivy.current_backend().valid_dtypes),
```

Comment:
Well, this line is too long as describe by the error message below.
ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py:15:89: E501 line too long (110 > 88 characters)